_component_design/assisted-search.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Assisted Search
+parent: Search and Filters
+layout: component
+category: Components
+usage: >
+  Use assisted search when dealing with a complex data set.  Assisted search provides users the ability to see available facets or filters within the search field, allowing for custom construction of search criteria before execution. 
+preview-image: preview-images/assisted-search.svg
+resource: true
+status: stable
+last-modified: 2017-11-07
+---
+
+{% include toc.html %}
+
+## Assisted search
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Default state
+
+Upon clicking the search box the user should be presented with a list of keyword that they can use to filter their search. The user can click or type out a keyword to use it as a filter to narrow down their search.
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Assist search specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/search-and-filters/search-assisted/search-assisted-default.svg)
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Loading state
+
+If autocomplete is taking time to load, a loading spinner should be used in place of results.
```

Comment:
Can this be quantified?
Try to use active voice.

Reply:
Use a loading spinner if results take longer than BLAH to load.

Reply:
Now reads, "Use a [loading spinner]({{site.baseurl}}/components/loading-indicators.html) if results take need to be processed prior to display."